cypress/integration/webapp/annotations.ts

```diff
@@ -0,0 +1,38 @@
+describe('Annotations', () => {
+  it('add annotation flow works as expected', () => {
+    cy.visit('/').wait(3000);
```

Comment:
Also don't wait an arbitrary amount of time (here 3000 ms). You most likely are waiting on a network request, which you can mock.

Eg https://github.com/pyroscope-io/pyroscope/blob/main/cypress/integration/webapp/serviceDiscovery.ts#L4-L9

Reply:
not sure it would work in this case because after getting response this page takes some time for rendering

Reply:
but let me research it better :)